sessions-core/src/test/java/com/adyen/checkout/sessions/core/internal/SessionInteractorTest.kt

```diff
@@ -707,6 +708,39 @@ internal class SessionInteractorTest(
         }
     }
 
+    @Nested
+    @DisplayName("when an update payment methods call is requested and")
```

Comment:
I think the test names need to be fixed

Reply:
Good catch!

Reply:
Yes but these tests are not about "when an update payment methods call is requested and" but about removing a stored payment methods

Reply:
Ahh I see. Totally missed that and confidently copied that part. 😅